examples/EvaluatingChatAssistant/EvaluatingChatAssistant.ipynb

```diff
@@ -0,0 +1,353 @@
+    "    content: string;\n",
+    "  }[]\n",
+    "}) {\n",
+    "  const client = wrapOpenAI(\n",
+    "    new OpenAI({\n",
+    "      baseURL: \"https://braintrustproxy.com/v1\",\n",
+    "      apiKey: process.env.OPENAI_API_KEY ?? \"\", // Can use OpenAI, Anthropic, Mistral etc. API keys here\n",
+    "    })\n",
+    "  )\n",
+    "\n",
+    "  const response = await client.chat.completions.create({\n",
+    "    model: \"gpt-4o\",\n",
+    "    messages: [\n",
+    "      {\n",
+    "        role: \"system\",\n",
+    "        content: \"You are a helpful and polite assistant who knows about sports.\"\n",
+    "      },\n",
+    "      ...chat_history,\n",
+    "      {\n",
+    "        role: \"user\",\n",
+    "        content: input,\n",
+    "      }\n",
+    "    ]\n",
+    "  });\n",
+    "  return response.choices[0].message.content || \"\";\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Scoring the eval\n",
+    "\n",
+    "We'll use a `Factuality` scoring function to check how the output of the completion compares to the expected value.\n",
+    "\n",
+    "For this cookbook, we will alter the [built in factuality prompt](https://github.com/braintrustdata/autoevals/blob/main/templates/factuality.yaml) slightly.\n",
+    "\n",
+    "(I have used the built-in Factuality scorer and it does not significantly reduce the score compared to running the eval with no chat history. If the model does not have enough context to factually answer the question and responds with a request for the user to be more specific, the Factuality scorer evaluates this response as having no factual inconsistencies between the expert and submitted answers. Because this cookbook is to illustrate how to incorporate chat history to produce a good response, we will tweak the built-in prompt slightly to account for this.)\n",
+    "\n",
+    "In the [altered factuality spec](https://github.com/braintrustdata/braintrust-cookbook/tree/main/examples/EvaluatingCustomerSupportAgent/assets/factuality.ts), you can see that we have added two more score choices:\n",
+    "- (F) The submitted answer asks for more context, specifics or clarification but provides factual information consistent with the expert answer.\n",
+    "- (G) The submitted answer asks for more context, specifics or clarification but does not provide factual information consistent with the expert answer.\n",
+    "\n",
+    "These will score (F) = 0.2 and (G) = 0 so the model gets some credit if there was any context it was able to gather from the user's input."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import { Eval } from \"braintrust\";\n",
+    "import dataset from \"./assets/dataset\"\n",
+    "import factualitySpec from \"./assets/factuality\";\n",
+    "import { LLMClassifierFromSpec } from \"autoevals\";\n",
```

Comment:
Maybe add some extra text in the text blob to describe/link to the docs for defining custom scorers and the `LLMClassifierFromSpec` function?